### PR TITLE
get settings from the store when popup shows

### PR DESCRIPTION
**Pull request by mytechtip**
also refactor LocalStore.setValue to save object

### CODE REVIEW

File: src/ts/background/store/settings/actions.ts [hunk 1]
```diff
@@ -3,19 +3,27 @@ import { LocalStore } from '../../../services/local-store'
...
-  CHANGE_COLOR: 'CHANGE_COLOR'
+  CHANGE_COLOR: 'CHANGE_COLOR',
+  GET_SETTINGS: 'GET_SETTINGS'
 }
 
 export function changeTheme (theme: string): AnyAction {
   return {
     type: ACTION_TYPES.CHANGE_THEME,
-    payload: LocalStore.setValue('theme', theme)
+    payload: LocalStore.setValue({ settings: { theme: theme } }, theme)
```
Comment (ttaoS) [inline src/ts/background/store/settings/actions.ts]: is this going to reset the whole setting?

Comment (mytechtip) [reply src/ts/background/store/settings/actions.ts]: yes, that was wrong, i've updated the implementation.

**Comment by mytechtip** [comment]
> `LocalStore.setValue(obj: any, value: any)` seems less intuitive than previous `LocalStore.setValue(key: string, value: any)`. Probably we need some comment on `setValue` function to explain what will be the key to retrieve the value?

I've reverted back the change.

File: src/ts/components/Welcome.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { Link } from 'react-router-dom'
+import { TERM_SERVICE_ROUTE } from '../constants/routes'
+import { IAppState } from '../background/store/all'
+import { connect } from 'react-redux'
+import { saveSettings } from '../background/store/settings'
+import { ThemeTypes } from './styles/themes'
+
+interface IWelcomeProp extends StateProps, DispatchProps {}
+
+interface IHomeState {
```
Comment (ttaoS) [inline src/ts/components/Welcome.tsx]: you may want to rename it to welcomexx

File: src/ts/components/Welcome.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            <Button onClick={this.handleChangeTheme}>Change them to light</Button>
+
+          </div>
+          <Link to={TERM_SERVICE_ROUTE}>Term</Link>
+        </LandingContainer>
+    )
+  }
+}
+
+const LandingContainer = styled('div')`
```
Comment (ttaoS) [inline src/ts/components/Welcome.tsx]: should we `welcomeContainer `?

Comment (fyang1024) [reply src/ts/components/Welcome.tsx]: The benefit is obviously it's more explicit. The downside is we may end up with long names. Do we suffix Container as XXXContainer and Simple Component as YYYComponent? UI libraries seem not doing that. e.g., semantic-ui-react just uses Button, Card, List, Grid, etc. 
I don't mind either way as long as we do that consistently

Comment (ttaoS) [reply src/ts/components/Welcome.tsx]: this container is HOC for style container within a component which i think should have the consistent prefix with the component name. It is different with component's `container`.

**Pull request merged**